global_costmap:
    global_frame: map
    robot_base_frame: base_link
    update_frequency: 5.0 # Hz
    static_map: true

    plugins:
    - {name: static_layer, type: "costmap_2d::StaticLayer"}
    - {name: inflation_layer, type: "costmap_2d::InflationLayer"}

    static_layer:
        map_topic: "/roomba/map_ready"
        track_unknown_space: true

    inflation_layer:
        inflation_radius: 0.31